Containerfile.ubi9

```
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/python-312-minimal:9.6-1760522101
+FROM registry.access.redhat.com/ubi9/python-312@sha256:d350329565030871c36d24c43533db869e892390fb47dc5c44512a32792dc623
```

I would prefer sticking with `-minimal`, the size difference is quite drastic.
```suggestion
FROM registry.access.redhat.com/ubi9/python-312:9.6-1762316373
```

---

Yes, I just updated to `9.6-1762316373`